hpolib/abstract_benchmark.py

```diff
@@ -171,6 +185,7 @@ def test(self, n_runs: int = 5, *args, **kwargs) -> Tuple[List, List]:
     @abc.abstractmethod
     def get_configuration_space() -> ConfigSpace.ConfigurationSpace:
         """ Defines the configuration space for each benchmark.
+        TODO: A random seed for the configspace!
```

Comment:
Can't you add this like it is already added in https://github.com/automl/HPOlib3/blob/master/hpolib/benchmarks/ml/xgboost_benchmark.py#L189?